mmv1/templates/terraform/encoders/gkehub2_feature.go.erb

```diff
@@ -0,0 +1,19 @@
+<%- # the license inside this block applies to this file
+        # Copyright 2023 Google Inc.
+        # Licensed under the Apache License, Version 2.0 (the "License");
+        # you may not use this file except in compliance with the License.
+        # You may obtain a copy of the License at
+        #
+        #     http://www.apache.org/licenses/LICENSE-2.0
+        #
+        # Unless required by applicable law or agreed to in writing, software
+        # distributed under the License is distributed on an "AS IS" BASIS,
+        # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        # See the License for the specific language governing permissions and
+        # limitations under the License.
+-%>
+// Set all the other fields to their short forms before forming url and setting ID.
```

Comment:
We may already have custom expander and flattener for same functionality. Would you mind check if templates/terraform/custom_expand/resource_from_self_link.go.erb and templates/terraform/custom_flatten/name_from_self_link.erb work in this case